fsspec/implementations/arrow.py

```diff
@@ -255,6 +255,7 @@ def __init__(
         port=0,
         user=None,
         kerb_ticket=None,
+        replication=3,
```

Comment:
Since it's a new kwarg, we had better have a docstring entry for it. Is it possible to set per-file at open(, mode="w") time?

Reply:
I add docstring for this parameter.

As far as I have searched, in the pyarrow implementation, this argument is only configurable by the constructor.